CHANGES_NEXT_RELEASE

```diff
@@ -2,4 +2,7 @@ Fix: modified engine version >=6 in packages.json
 Fix: update logops dependency from 1.0.8 to 2.1.0
 Fix: user provider timestamp correctly propagated to mapped entities when multientity pluging is used (#748)
 Add: new functions to set callbacks for removal of devices and groups (#735)
-Fix: add ?type parameter in CB request updates to avoid potential entity ambiguities (#733)
\ No newline at end of file
+Fix: add ?type parameter in CB request updates to avoid potential entity ambiguities (#733)
+Add: support HTTPS requests toward Keystone (if url is used in the authorization configuration)
+Add: support OAuth2 provider as authentication backend (#591)
+Add: support for authentication to NGSI subscription requests (#592) 
```

Comment:
These of the new lines are already mentioned in the [release notes of version 2.7.0](https://github.com/telefonicaid/iotagent-node-lib/releases/tag/2.7.0). I understand this PR (in fact, the old #600) covers only issue #592...

Maybe a merge conflict was incorrectly solved so these two lines as missappear here?

Reply:
In addition, with regards to:

```
Add: support for authentication to NGSI subscription requests
```

I undertand that NGSI refers to NGSIv1 **and** NGSIv2. Fine with it, just checking (given that parent issue #592 only mentions NGSIv1 in the title).

Reply:
Fixed with cdef73a623a9985bbb59d61b8ba144aa8fab48b2

Reply:
>I undertand that NGSI refers to NGSIv1 and NGSIv2. Fine with it, just checking (given that parent issue #592 only mentions NGSIv1 in the title).

You're right! :)